api/DedicatedWorkerGlobalScope.json

```diff
@@ -0,0 +1,332 @@
+          },
+          "opera": {
+            "version_added": "10.6"
+          },
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "4"
+          },
+          "safari_ios": {
+            "version_added": "5.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "ononline_onoffline": {
+        "__compat": {
+          "description": "<code>ononline</code>, <code>onoffline</code>",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "29"
+            },
+            "firefox_android": {
+              "version_added": "29"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "console": {
+        "__compat": {
+          "description": "<code>console</code>",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "29",
+                "version_removed": "30",
+                "alternative_name": "WorkerConsole"
+              },
+              {
+                "version_added": "30"
+              }
+            ],
+            "firefox_android": {
+              "version_added": "29"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          }
+        }
+      },
+      "performance": {
```

Comment:
Looks like this is actually on WorkerGlobalScope: https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/performance
Let's remove this from this table.

Reply:
👍